service/src/main/java/uk/gov/hmcts/reform/fpl/service/orders/generator/ManageOrdersClosedCaseFieldGenerator.java

```diff
@@ -27,13 +28,13 @@
 
         Map<String, Object> data = new HashMap<>();
 
-        if (IsFinalOrder.YES == order.getFinalOrder()) {
+        if (Order.isOrderFinal(order)) {
```

Comment:
Just a thought. Final could be a boolean attribute of the Order.
This way, instead of calling the static method `Order.isOrderFinal(order)`, you could just call `order.isOrderFinal()`.